src/taipy/templates/taipy-default-template/hooks/post_gen_project.py

```diff
@@ -10,8 +10,174 @@
+        elif "core = Core()" in main_content:
+            main_file.write('    tp.run(gui, core, title="{{cookiecutter.__application_title}}")\n')
+        else:
+            main_file.write('    gui.run(title="{{cookiecutter.__application_title}}")\n')
+
+
+def handle_single_page_app():
+    shutil.rmtree(os.path.join(os.getcwd(), "pages"))
+
+    with open(os.path.join(os.getcwd(), "sections", "main.txt"), "a") as main_file:
+        main_file.write("\n")
+        main_file.write("    gui = Gui(page=page)\n")
+
+    handle_run_service()
+
+    with open(os.path.join(os.getcwd(), "sections", "page_content.txt"), "a") as page_content_file:
+        page_content_file.write(
+            '''
+\npage = """
+<center>
+<|navbar|lov={[("home", "Homepage"), ("https://docs.taipy.io/en/latest/manuals/about/", "Taipy Docs"), ("https://docs.taipy.io/en/latest/getting_started/", "Getting Started")]}|>
+</center>
+
+<|
+<center>
+    <|{logo}|image|height=200px|width=200px|on_action=image_action|>
+</center>
+|>
+
+# Taipy Application
+
+"""
+'''
+        )
+
+
+def handle_multi_page_app(pages):
+    number_of_pages = int(pages[0])
+
+    for page_i in range(1, number_of_pages + 1):
+        try:
+            pages[page_i] = pages[page_i] or "page_" + str(page_i)
+        except IndexError:
+            page_name = "page_" + str(page_i)
+            pages.append(page_name)
+    pages = pages[1 : number_of_pages + 1]
+
+    for page_name in pages:
+        os.mkdir(os.path.join(os.getcwd(), "pages", page_name))
+        with open(os.path.join(os.getcwd(), "pages", "page_example", "page_example.md"), "r") as page_md_file:
+            page_md_content = page_md_file.read()
+        page_md_content = page_md_content.replace("Page example", page_name.replace("_", " ").title())
+        with open(os.path.join(os.getcwd(), "pages", page_name, page_name + ".md"), "w") as page_md_file:
+            page_md_file.write(page_md_content)
+
+        with open(os.path.join(os.getcwd(), "pages", "page_example", "page_example.py"), "r") as page_content_file:
+            page_py_content = page_content_file.read()
+        page_py_content = page_py_content.replace("page_example", page_name)
+        with open(os.path.join(os.getcwd(), "pages", page_name, page_name + ".py"), "w") as page_content_file:
+            page_content_file.write(page_py_content)
+
+    with open(os.path.join(os.getcwd(), "pages", "__init__.py"), "a") as page_init_file:
+        for page_name in pages:
+            page_init_file.write(f"from .{page_name}.{page_name} import {page_name}\n")
+
+    shutil.rmtree(os.path.join(os.getcwd(), "pages", "page_example"))
+
+    newline = ",\n\t"
+    user_page_dict = newline.join(f'"{page_name}": {page_name}' for page_name in pages)
+    page_dict = """
+\npages = {
+    "/": root_page,
+    "Taipy-Docs": "https://docs.taipy.io/en/latest/manuals/about/",
+    "Getting-Started": "https://docs.taipy.io/en/latest/getting_started/",
+    {pages}
+}
```

Comment:
I would expect to have in all the empty pages generated. I would also remove the doc and the getting started pages.

Reply:
The `{pages}` should be all the empty pages generated.

About the doc and getting started link, I would like to keep those, because if the user just starts using taipy, a link to the doc and some tutorial should be helpful.